src/components/AgencySearchResults.js

```diff
@@ -37,6 +46,8 @@ const AgencySearchResults = ({ data, groupKey, groupValues, onClick }) => {
     data: noFederal.filter(d => d[groupKey] === null),
   })
 
+  const usState = oriToState(noFederal[0].ori)
```

Comment:
i think we should pass `usState` down as a prop rather than parse and lookup from ORIs (since it's available in the `LocationFilter` component)